lib/debezium/types_bench_test.go

```diff
@@ -19,7 +19,7 @@ func BenchmarkDecodeDecimal_P64_S10(b *testing.B) {
 		assert.NoError(b, err)
 		dec, err := field.DecodeDecimal(bytes)
 		assert.NoError(b, err)
-		assert.Equal(b, "123456789012345678901234567890123456789012345678901234.1234567889", dec.Value())
+		assert.Equal(b, "123456789012345678901234567890123456789012345678901234.1234567890", dec.String())
```

Comment:
This was decoding correctly, the unscaled value is `1234567890123456789012345678901234567890123456789012341234567890`

Reply:
Which makes sense since this number is just `1234567890` repeating (with a 1234 before the decimal point).